src/modules/campath/mod.rs

```diff
@@ -162,6 +176,17 @@ fn force_live(marker: MainThreadMarker) {
     IS_FORCED.set(marker, true);
 }
 
+fn rotate_round_z(point: glam::Vec3, rad: f32) -> glam::Vec3 {
```

Comment:
rotate_about_z?

Reply:
it is supposed to be "rotate_around_z" but I think I forget an "a" there.